tests/functional/pages/landing.page.js

```diff
@@ -10,6 +10,26 @@ class LandingPage extends BasePage {
     return '.qa-btn-get-started';
   }
 
+  useOfInformation() {
```

Comment:
Why aren't we using an `introduction.page.js`?

Reply:
Arent' the landing/introduction pages the same thing?

Reply:
Yes really but since we don't have a concept of a landing page I would just rename `landing.page.js` to `introduction.page.js` to make it clear.
Don't forget to change it here as well `super('landing-page');`